friture/audioproc.py

```diff
@@ -60,9 +57,9 @@ def decimate(self, samples):
         if self.decimation > 1:
             samples.shape = len(samples) // self.decimation, self.decimation
             # the full way
-            # samples = samples.mean(axis=1)
-            # the simplest way
-            samples = samples[:, 0]
+            samples = samples.mean(axis=1)
+            # the fastest way
```

Comment:
(I didn't actually profile this, idk how much computation time goes here vs actually doing the fft and rendering)

Reply:
hm, I just profiled (with a value of self.decimation == 16) and with the faster way, the decimation function takes about 7.05% of a the FFT's duration, and with the slower way it takes about 93.1% of the FFT's duration. So, a pretty significant cost, assuming the FFT is the expensivest part of this process.